src/main/webapp/app/feature-overview/feature-overview.component.html

```diff
@@ -0,0 +1,124 @@
+<head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
```

Comment:
Wait, I think it is better if we host this ourselves! Don't we have FA included already?

Reply:
yes, please use the local version.
It's really `bad practice` to link to external URLs like this

Reply:
see 6d76e007b13b0e7ef7f2599e75802b607d7d5078 & 00b23dde1932cd3def02faff1e6ac637e80c86bc